<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-41G76H2" failures="1" tests="7" name="REST_Assure_Get_Call" time="6.384" errors="0" timestamp="12 Mar 2019 10:47:53 GMT">
  <testcase classname="com.qa.test.REST_Assure_Get_Call" name="getResponseBody" time="2.179"/>
  <testcase classname="com.qa.test.REST_Assure_Get_Call" name="validate_get_all_headers" time="0.609"/>
  <testcase classname="com.qa.test.REST_Assure_Get_Call" name="validate_get_incorrect_response_status_code" time="1.193"/>
  <testcase classname="com.qa.test.REST_Assure_Get_Call" name="validate_get_incorrect_response_status_line" time="0.574"/>
  <testcase classname="com.qa.test.REST_Assure_Get_Call" name="validate_get_response_status_code" time="0.577"/>
  <testcase classname="com.qa.test.REST_Assure_Get_Call" name="validate_get_response_status_line" time="0.582"/>
  <testcase classname="com.qa.test.REST_Assure_Get_Call" name="validate_response_headers_using_lamda_expression" time="0.575">
    <failure type="java.lang.AssertionError" message="Header doesn&amp;apos;t exists expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Header doesn't exists expected [true] but found [false]
at com.qa.test.REST_Assure_Get_Call.lambda$validate_response_headers_using_lamda_expression$1(REST_Assure_Get_Call.java:73)
at java.base/java.lang.Iterable.forEach(Iterable.java:75)
at com.qa.test.REST_Assure_Get_Call.validate_response_headers_using_lamda_expression(REST_Assure_Get_Call.java:71)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- validate_response_headers_using_lamda_expression -->
</testsuite> <!-- REST_Assure_Get_Call -->
